mta-archive-builder/README.md

```diff
@@ -0,0 +1,42 @@
+# Multitarget Application Archive Builder
+The multitarget application archive builder is a command-line tool that packages a multitarget application into a deployable archive (MTAR). For full documentation please visit the [SAP Help Portal](https://help.sap.com/viewer/58746c584026430a890170ac4d87d03b/Cloud/en-US/ba7dd5a47b7a4858a652d15f9673c28d.html).
+
+This image can be used to build SAP Multitarget Applications (MTA) containing Java and Node.js modules.
+
+# How to use this image
+
+On a linux machine you can run 
+
+`docker run --rm -v `pwd`:/project -it mta-archive-builder:latest mtaBuild --version`
```

Comment:
Why `-it`? The example should run without it.

```suggestion
`docker run --rm mta-archive-builder mtaBuild --version`
```

Reply:
I think `-it` is not strongly required